configs/src/deployments/types.ts

```diff
@@ -64,3 +64,12 @@ export enum ContractNames {
   TOKEN_VOTING_REPO_IMPLEMENTATION = 'TokenVotingRepoImplementation',
   ENS_REGISTRY = 'ENSRegistry',
 }
+
+export type NetworkDomain = {
+  daoEns: string;
+  pluginEns: string;
+};
+
+export type NetworkDomains = {
+  [network: string]: NetworkDomain;
+};
```

Comment:
Nice, but i am wondering if we can have a better naming here?
`OsxENSDomain` instead of `NetworkDomain`
`NetworksOSxENSDomains` instead of `NetworkDomains`

or

`ENSNetworkDomain` & `ENSNetworkDomainsMap`

Reply:
Hmmm. 
I like `ENSNetworkDomain` & `ENSNetworkDomainsMap`.

Reply:
`osxEnsDomain` and then we use it `osxEnsDomain[network]`? 

Can we use the osx-commons-config network types instead of `string`?